modules/data-entry/src/main/frontend/src/dataHomepage/Filters.jsx

```diff
@@ -194,6 +194,7 @@ function Filters(props) {
   let handleChangeFilter = (index, path) => {
 
     // Load up the comparators for this index, if not already loaded
+    // eslint-disable-next-line no-unused-vars
     let [loadedComparators, component] = getOutputChoices(path);
```

Comment:
If `component` is not used, this could just be:


```suggestion
    let [loadedComparators] = getOutputChoices(path);
```